org.openmw.OpenMW.json

```diff
@@ -0,0 +1,272 @@
+{
+  "app-id": "org.openmw.OpenMW",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openmw-launcher",
+  "rename-appdata-file": "openmw.appdata.xml",
+  "rename-desktop-file": "openmw.desktop",
+  "finish-args": [
+    "--share=ipc", "--socket=x11", "--device=dri",
+    "--socket=pulseaudio"
+  ],
+  "build-options" : {
+    "cflags": "-O2 -g",
+    "cxxflags": "-O2 -g"
+  },
+  "cleanup": ["/include", "/lib/pkgconfig",
+    "/share/pkgconfig", "/share/aclocal",
+    "/share/doc",
+    "/man", "/share/man", "/share/gtk-doc",
+    "/share/vala",
+    "*.la", "*.a"],
+    "modules": [
+      {
+        "name": "ffmpeg",
+        "config-opts": [
+          "--disable-everything",
+          "--disable-static",
+          "--enable-shared",
+          "--disable-programs",
+          "--disable-doc",
+          "--disable-avdevice",
+          "--disable-avfilter",
+          "--disable-postproc",
+
+          /* Enough to run the base game, might need extending for mod content */
+          "--enable-decoder=bink,binkaudio_rdft,mp3,pcm_s16le,vorbis",
+          "--enable-demuxer=bink,mp3,ogg,pcm_s16le,wav"
+        ],
+        "sources": [
+          {
+            "type": "archive",
+            "url": "http://ffmpeg.org/releases/ffmpeg-3.4.tar.xz",
```

Comment:
I'd rather use 3.3.x, at least in Arch we're having some issues with 3.4 across packages.